ivy_tests/test_ivy/test_frontends/test_paddle/test_search.py

```diff
@@ -148,6 +148,37 @@ def test_paddle_argsort(
     )
 
 
+@handle_frontend_test(
+    fn_tree="paddle.index_sample",
+    array_indices_axis=helpers.array_indices_axis(
+        array_dtypes=helpers.get_dtypes("valid"),
+        indices_dtypes=helpers.get_dtypes("integer"),
+        min_num_dims=2,
+        max_num_dims=2,
```

Comment:
the array and axis generation strategy also seems quite restrictive to me. The input `x` in [paddle.index_select](https://github.com/PaddlePaddle/Paddle/blob/release/2.5/python/paddle/tensor/search.py#L312) can be any n-dim tensor so there's no need to limit it to 2D. 
You'll also need to add the generation strategy for the `axis` parameter which can be in the range of `[-dim, dim-1]` where `dim` is the dimension of the input `x`